sat/cli/bootsys/filesystems.py

```diff
@@ -292,6 +293,7 @@ def do_ceph_unmounts(ssh_client, hostname):
     inflector = engine()
 
     try:
+        ssh_client.set_missing_host_key_policy(AutoAddPolicy())  # Set AutoAddPolicy
```

Comment:
If we are already setting the missing host key policy in the get_ssh_client function, then we should not need to do it again here. The `do_ceph_unmounts` function is called by `do_mgmt_shutdown_power`. That's where the `SSHClient` instance is created by a call to `get_ssh_client`, and that function already calls `set_missing_host_key_policy` on the `ssh_client`.